gui/wxpython/core/settings.py

```diff
@@ -808,8 +808,7 @@ def _internalSettings(self):
         self.internalSettings["appearance"]["gSelectPopupHeight"]["max"] = 1000
         self.internalSettings["appearance"]["commandNotebook"]["choices"] = (
             _("Basic top"),
-            _("Basic left"),
-            _("Fancy green"),
+            _("Normal style"),
             _("List left"),
```

Comment:
I wanted to remove the Fancy green, not Basic left, what is supposed to be Normal style?